### PR TITLE
thouroughly test choice service, change choices model to programChoice, create basic set/get for user service

### PR DESCRIPTION
**Pull request by lucy-fan**
please take a look at mostly choices.service.ts again, changes were minor in terms of structure but there were some silly bugs that have all been fixed so functionality should be a go!

### CODE REVIEW

**Comment by lucy-fan** [comment]
i also added in basic functionality for the user service inside user.service.ts, unsure what other functions the user service needs other than passing information. if that's sufficient, then the user service is also completed!

File: application-portal/src/app/services/user/user.service.ts [hunk 1]
```diff
@@ -1,9 +1,113 @@
...
+
+  setUser(user: User) {
+    this.user = user;
+  }
+
+  getId() {
+    return this.user.id;
+  }
+
+  // not sure when we would use this? but i included it
```
Comment (khannotations) [inline application-portal/src/app/services/user/user.service.ts]: Please remove this method

File: application-portal/src/app/services/user/user.service.ts [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+  setId(id: number) {
+    this.user.id = id;
+    this.id = id;
+  }
+
+  getFirstName() {
```
Comment (khannotations) [inline application-portal/src/app/services/user/user.service.ts]: Are all these User methods necessary? If you're not using them then you don't need to write them :)

**Comment by lucy-fan** [comment]
after some thought, i realized that we never really need to modify the user info, so i removed all the set methods. new user service has been pushed to my branch and should be ready to merge with master!

**Pull request merged**